app/web/src/components/IdeToolbarNew/IdeToolbarNew.js

```diff
@@ -5,7 +5,10 @@ import { useIdeState, codeStorageKey } from 'src/helpers/hooks/useIdeState'
 import { copyTextToClipboard } from 'src/helpers/clipboard'
 import { requestRender } from 'src/helpers/hooks/useIdeState'
 import { encode, decode } from 'src/helpers/compress'
-import { flow } from 'lodash/fp'
+import { flow, identity } from 'lodash/fp'
+import { fileSave } from 'browser-fs-access'
```

Comment:
nice library that abstracts a bunch browser compatibility issues with saving files from the browser.
more context : https://github.com/zalo/CascadeStudio/pull/39#issuecomment-766911556